lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -144,6 +151,10 @@ def credit_request(money, payment_method, options)
         commit('credit', build_authorization_request(money, payment_method, options), :ok, 'SENT_FOR_REFUND', options)
       end
 
+      def store_request(credit_card, options)
+        commit('store', build_store_request(credit_card, options), options)
```

Comment:
Note that we are intentionally not passing the `*success_requirements` argument. It is optional, and not applicable to `store`. See comments RE: `success_and_message_from`.